src/Adliance.AspNetCore.Buddy.OpenTelemetry/Extensions/BuddyApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,33 @@
+using System.Reflection;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.FileProviders;
+
+namespace Adliance.AspNetCore.Buddy.OpenTelemetry.Extensions;
+
+public static class BuddyApplicationBuilderExtensions
+{
+    /// <summary>
+    /// Adds a static file server for /otel-js/telemetry.js.
+    /// </summary>
+    /// <param name="applicationBuilder">The application builder.</param>
+    /// <returns>The application builder to allow further calls.</returns>
+    public static IApplicationBuilder AddBuddyOpenTelemetryBrowserAssets(
```

Comment:
I think the extension for IApplicationBuilder should be named "Use" (for adding middleware) and not "Add" (for adding to DI container).